bug: panic on aws_security_group rule

**Version**
```sh
$ tfsec --version
v1.0.2
```

**Describe the bug**
tfsec panics with:
```
panic: WithoutOptionalAttributesDeep does not support the given type
```
(full trace below)
on an `aws_security_group` that have a `cidr_block` defined with a ternary operator when one of the possible value comes from a data specified in a parent module (quite specific :sweat_smile: )

**To Reproduce**

Steps to reproduce the behavior:
1. Create a `test.tf` file with: 
```sh
$ cat > test.tf <<EOF
data "aws_vpc" "default" {
  default = true
}

module "test" {
  source     = "./sub/"
  cidr_block = data.aws_vpc.default.cidr_block
}
EOF
```
2. Create the module `sub` with

```sh
$ mkdir sub
$ cat > sub/main.tf <<EOF                                                                                                                                                                                                          
variable "cidr_block" {}

variable "open" {                
  default = false
}                

resource "aws_security_group" "this" {
  name = "Test"                       

  ingress {    
    description = "HTTPs"
    from_port   = 443    
    to_port     = 443
    protocol    = "tcp"
    self        = ! var.open
    cidr_blocks = var.open ? [var.cidr_block] : null
  }                                                 
}  
EOF
```

3. Then init and run tfsec
```sh
$ terraform init
$ tfsec
panic: WithoutOptionalAttributesDeep does not support the given type

goroutine 1 [running]:
github.com/zclconf/go-cty/cty.Type.WithoutOptionalAttributesDeep({{0x0, 0x0}})
	/home/runner/work/tfsec/tfsec/vendor/github.com/zclconf/go-cty/cty/type.go:152 +0x3de
github.com/zclconf/go-cty/cty.Type.WithoutOptionalAttributesDeep({{0xd14608, 0xc0004270c8}})
	/home/runner/work/tfsec/tfsec/vendor/github.com/zclconf/go-cty/cty/type.go:135 +0x457
github.com/zclconf/go-cty/cty/convert.getConversion.func1({{{0xd144f0, 0x12eaa70}}, {0x0, 0x0}}, {0x0, 0x0, 0x0})
	/home/runner/work/tfsec/tfsec/vendor/github.com/zclconf/go-cty/cty/convert/conversion.go:43 +0x285
github.com/zclconf/go-cty/cty/convert.retConversion.func1({{{0xd144f0, 0x12eaa70}}, {0x0, 0x0}})
	/home/runner/work/tfsec/tfsec/vendor/github.com/zclconf/go-cty/cty/convert/conversion.go:199 +0x3a
github.com/hashicorp/hcl/v2/hclsyntax.(*ConditionalExpr).Value(0xc000417e30, 0xc000426b10)
	/home/runner/work/tfsec/tfsec/vendor/github.com/hashicorp/hcl/v2/hclsyntax/expression.go:693 +0xf84
github.com/aquasecurity/tfsec/internal/pkg/block.getStrings({0xd14058, 0xc000417e30}, 0x10)
	/home/runner/work/tfsec/tfsec/internal/pkg/block/attribute.go:254 +0x208
github.com/aquasecurity/tfsec/internal/pkg/block.(*Attribute).ValueAsStrings(...)
	/home/runner/work/tfsec/tfsec/internal/pkg/block/attribute.go:243
github.com/aquasecurity/tfsec/internal/pkg/adapter/aws/vpc.adaptSGRule(0xc000422c80, {0xc00044d3a0, 0x2, 0x2})
	/home/runner/work/tfsec/tfsec/internal/pkg/adapter/aws/vpc/adapt.go:142 +0x28a
github.com/aquasecurity/tfsec/internal/pkg/adapter/aws/vpc.(*sgAdapter).adaptSecurityGroup(0xc00035b7b0, 0xc0004230e0, {0xc00044d3a0, 0x2, 0x2})
	/home/runner/work/tfsec/tfsec/internal/pkg/adapter/aws/vpc/adapt.go:96 +0x15c
github.com/aquasecurity/tfsec/internal/pkg/adapter/aws/vpc.(*sgAdapter).adaptSecurityGroups(0xc00035b7b0, {0xc00044d3a0, 0x2, 0x2})
	/home/runner/work/tfsec/tfsec/internal/pkg/adapter/aws/vpc/adapt.go:45 +0x11b
github.com/aquasecurity/tfsec/internal/pkg/adapter/aws/vpc.Adapt({0xc00044d3a0, 0x2, 0x2})
	/home/runner/work/tfsec/tfsec/internal/pkg/adapter/aws/vpc/adapt.go:16 +0x156
github.com/aquasecurity/tfsec/internal/pkg/adapter/aws.Adapt({_, _, _})
	/home/runner/work/tfsec/tfsec/internal/pkg/adapter/aws/adapt.go:76 +0xe1f
github.com/aquasecurity/tfsec/internal/pkg/adapter.Adapt({0xc00044d3a0, 0x2, 0x2})
	/home/runner/work/tfsec/tfsec/internal/pkg/adapter/adapt.go:19 +0x4e
github.com/aquasecurity/tfsec/internal/pkg/scanner.(*Scanner).Scan(0xc000460240, {0xc00044d3a0, 0x0, 0x2})
	/home/runner/work/tfsec/tfsec/internal/pkg/scanner/scanner.go:51 +0x9f
github.com/aquasecurity/tfsec/internal/app/tfsec/cmd.glob..func1(0x127f080, {0x12eaa70, 0x0, 0x0})
	/home/runner/work/tfsec/tfsec/internal/app/tfsec/cmd/root.go:114 +0x7ec
github.com/spf13/cobra.(*Command).execute(0x127f080, {0xc00001e200, 0x0, 0x0})
	/home/runner/work/tfsec/tfsec/vendor/github.com/spf13/cobra/command.go:856 +0x60e
github.com/spf13/cobra.(*Command).ExecuteC(0x127f080)
	/home/runner/work/tfsec/tfsec/vendor/github.com/spf13/cobra/command.go:974 +0x3bc
github.com/spf13/cobra.(*Command).Execute(...)
	/home/runner/work/tfsec/tfsec/vendor/github.com/spf13/cobra/command.go:902
main.main()
	/home/runner/work/tfsec/tfsec/cmd/tfsec/main.go:11 +0x25
```

**Additional context**
`tfsec` does not crash anymore if:
* I add parenthesis  around the ternary operator:
```hcl
cidr_blocks = (var.open ? [var.cidr_block] : null)
```
* `cidr_block` is defined with a string value directly in the module call:
```
module "test" {
  source     = "./sub/"
  cidr_block = "plop"
}
```
* I execute `tfsec` directly in the `sub` directory

--- a/go.mod
+++ b/go.mod
@@ -19,7 +19,7 @@ require (
 	github.com/owenrumney/squealer v0.3.1
 	github.com/spf13/cobra v1.3.0
 	github.com/stretchr/testify v1.7.0
-	github.com/zclconf/go-cty v1.10.0
+	github.com/zclconf/go-cty v1.8.0
 	github.com/zclconf/go-cty-yaml v1.0.2
 	golang.org/x/crypto v0.0.0-20210817164053-32db794688a5
 	golang.org/x/text v0.3.7

